invokeai/frontend/web/src/features/gallery/components/ImageGrid/GalleryImage.tsx

```diff
@@ -1,3 +1,4 @@
+import { Text } from '@chakra-ui/react';
```

Comment:
Please import from `@invokeai-ui/ui-library`

Reply:
is it **@invoke-ai/ui-library** or **@invokeai-ui/ui-library**